pkg/Cpanel/Security/Advisor/Assessors/Kernel.pm

```diff
@@ -29,22 +29,142 @@ package Cpanel::Security::Advisor::Assessors::Kernel;
 use strict;
 use base 'Cpanel::Security::Advisor::Assessors';
 use Cpanel::SafeRun::Errors ();
+use Cpanel::JSON            ();
 use Cpanel::Kernel          ();
 use Cpanel::OSSys::Env      ();
+use Cpanel::Version         ();
+use Cpanel::RPM             ();
+use Cpanel::Logger          ();
+use Cpanel::DIp::MainIP     ();
+use Cpanel::NAT             ();
+use HTTP::Tiny              ();
+
+our $KC_CP_VERSION = q{11.63};
+our $KC_VERIFY_URL = q{https://verify.cpanel.net};
+our $KC_M2_URL     = q{manage2.cpanel.net};
+if ( Cpanel::Logger::is_sandbox() ) {
+    $KC_VERIFY_URL = q{http://cody.verify.manage.devel.cpanel.net};
```

Comment:
Are we sure that this box is going to continue to exist for the life of our sandboxes?  What happens if this person leaves?